lms/djangoapps/ccx/utils.py

```diff
@@ -528,3 +528,23 @@ def multiple_ccx_per_coach(course):
             False,
         )
     )
+
+
+def is_staff_allowed_to_access_ccx_coach_tab(user, ccx_id):
```

Comment:
Isn't it preferable to carry out this function in course_operations and just define the varible for the view here? that way we only use the run_extension_point.